docs/pages/reference/api/introduction.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Teleport API Introduction
+description: Introduction for the Teleport API
+---
+
+The Teleport Auth API provides a gRPC API for remotely interacting with a Teleport Auth server.
+
+We recently created a public [Go client](https://pkg.go.dev/github.com/gravitational/teleport/api/client)
```

Comment:
Let's remove `we recently`. This article will stay evergreen for a while, so this will become not relevant in a year.